### PR TITLE
#157505109 Add SQLAlchemy Hook

### PR DESCRIPTION
**Pull request by ibutiti**
#### What does this PR do?
Adds and configures `flask-sqlalchemy` for the project.

##### Why this is needed
Activo will use SQLAlchemy as the ORM tool and this chore adds it to the project

##### Acceptance Criteria
The Activo api project having SQLAlchemy added with one model (user) and an initial database upgrade script. 

##### Dev Notes
The database to connect here is PostgreSQL
User model attributes
-name
-email
-image_url
The `user` model should be defined inside the `models` folder. 

#### How should this be manually tested?
Some configuration to be done:
1. Create a `.env` file in your project root
2. Add the following to the `.env` file:
```
FLASK_ENV=development
SQLALCHEMY_DATABASE_URI=postgresql://{your db username}:{your db password}@localhost/{your db name}'
```
3. Note that the `FLASK_ENV` variable can take any one of the values `development`, `testing` or `production` and will load the corresponding configuration as set in the `config.py` file.
4. Run migrations by executing the command `flask db upgrade`.
5. At this point, the app can be run by executing the command `python manage.py`.
6. Open your browser and navigate to `127.0.0.1:5000`. You should see the text `{message="Welcome to the AM api"}`

#### Any background context you want to provide?
Switched from `flask-script` to the `flask-cli` tools pre-bundled with flask.

#### What are the relevant pivotal tracker stories?
[#157505109](https://www.pivotaltracker.com/story/show/157505109)

#### Screenshots (if appropriate)
N/A

#### Questions:
None

### CODE REVIEW

**Comment by ibutiti** [comment]
Hey @SEUNAGBEYE, I think this model is intended as a placeholder for sqlalchemy setup to ensure that the setup is done well and we're at least able to run 1 migration.

File: config.py [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+
     DEBUG = True
 
+
 class TestingConfig(Config):
-    """App testing configuration"""
+    """App testing configuration."""
+
     TESTING = True
+    SQLALCHEMY_DATABASE_URI = getenv('TEST_DATABASE_URI', default='postgresql://localhost/activo_test')
```
[the remark this answers is not in the document]
Comment (ibutiti) [reply config.py]: It is possible @kodero, though I've seen a lot of guides recommending using the same db as production to run tests

[the remark this answers is not in the document]
Comment (kodero) [reply config.py]: Alright, no biggie. Let's leave it that way.

**Comment by ibutiti** [comment]
@kodero, PR ready for merge.

**Pull request merged**